src/component/misc/validation/InputValidationMessage.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useI18n } from "../../hook/useI18n";
+import ValidationResult, {
+  mapToCssClass,
+  Severity,
+} from "../../../model/form/ValidationResult";
+import "./InputValidationMessage.scss";
+
+interface InputValidationMessageProps {
+  message: ValidationResult;
+}
+
+const InputValidationMessage: React.FC<InputValidationMessageProps> = (
+  props
+) => {
+  const { message } = props;
+  const { i18n } = useI18n();
+  const tooltip =
```

Comment:
This looks strange

![image](https://user-images.githubusercontent.com/1140626/123287567-19435180-d50f-11eb-9058-b324a7c26d1d.png)

Reply:
Fixed.